src/cares_wrap.cc

```diff
@@ -618,6 +620,9 @@ class QueryWrap : public AsyncWrap {
                  int dnsclass,
                  int type) {
     channel_->EnsureServers();
+    TRACE_EVENT_NESTABLE_ASYNC_BEGIN1(
+      "node,node.dns", trace_name_, this,
```

Comment:
Let's make the category on these `node.dns.native`. Once we have the V8 trace instrinsic landed they will be paired up with the `node.dns.js` side.

Also, there's a macro to make the naming easier... e.g. `TRACING_CATEGORY_NODE2(dns, native)` expands to `node,node.dns,node.dns.native` to ensure we have a proper hierarchy.